.github/workflows/ci.yml

```diff
@@ -0,0 +1,172 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  pull_request_target:
+    branches: [ master ]
+    types: [ labeled ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: |
+      github.event_name != 'pull_request_target' &&
```

Comment:
This is to avoid running tests twice for every pull request, right?

Reply:
@kevinbarabash Yes, `test` doesn't require secrets so it can be run on `pull_request` event.